model/perf_data.go

```diff
@@ -6,6 +6,20 @@ import (
 	"github.com/evergreen-ci/sink"
 	"github.com/mongodb/grip"
 	"github.com/montanaflynn/stats"
+	"github.com/pkg/errors"
+	mgo "gopkg.in/mgo.v2"
+	"gopkg.in/mgo.v2/bson"
+)
+
+const (
+	idField           = "_id"
+	rollupsField      = "rollups"
+	rollupNameField   = "rollups.name"
+	rollupVerField    = "rollups.version"
+	newRollupVal      = "rollups.$.val"
+	newRollupVer      = "rollups.$.version"
+	pushNewRollup     = "$push"
+	setExistingRollup = "$set"
```

Comment:
I think we should probably not store constants for the dotted field names and just assemble them at runtime. I think the _id field is probably stored in a var elsewhere, and I'm sort of on the fence about storing $ operators in constants because it probably makes the code harder to read, but I could be convinced.